src/protagonist/Engine/Ingest/Timebased/Transcode/ElasticTranscoder.cs

```diff
@@ -76,24 +78,29 @@ public class ElasticTranscoder : IMediaTranscoder
     {
         var asset = context.Asset;
         var assetId = context.AssetId;
-        var technicalDetails = asset.FullImageOptimisationPolicy.TechnicalDetails;
-        var outputs = new List<CreateJobOutput>(technicalDetails.Length);
+        var timeBasedPolicies = asset.ImageDeliveryChannels.Where(i => i.Channel == AssetDeliveryChannels.Timebased)
+            .Select(x => JsonSerializer.Deserialize<List<string>>(x.DeliveryChannelPolicy.PolicyData))
+            .SelectMany(i => i).ToList();
+        var outputs = new List<CreateJobOutput>();
 
-        foreach (var technicalDetail in technicalDetails)
+        foreach (var timeBasedPolicy in timeBasedPolicies)
         {
             var mediaType = context.Asset.MediaType;
-            var (destinationPath, presetName) =
-                TranscoderTemplates.ProcessPreset(mediaType, assetId, technicalDetail, jobId);
-
+            
             // TODO - handle empty path/presetname
-            var mappedPresetName = settings.DeliveryChannelMappings.TryGetValue(presetName, out var mappedName)
-                ? mappedName
-                : presetName;
+            if (!settings.DeliveryChannelMappings.TryGetValue(timeBasedPolicy, out var mappedPresetName))
+            {
+                logger.LogWarning("Unable to find preset {TimeBasedPolicy} in the allowed mappings", timeBasedPolicy);
+                continue;
+            }
+            
+            var destinationPath = TranscoderTemplates.ProcessPreset(
+                mediaType, assetId, jobId, timeBasedPolicy.Split('-')[1]);

```

Comment:
`timeBasedPolicy.Split('-')[1]` feels loose and isn't immediately clear what it's doing.

Is it worth having a construct that represents one of our presets, exposing `.Format` property? Or having a `timeBasedPolicy.GetFormat()` extension method to make it easier to comprehend at a skim read?